scripts/newinstall.sh

```diff
@@ -27,10 +27,10 @@ LSST_EUPS_USE_EUPSPKG=${LSST_EUPS_USE_EUPSPKG:-true}
 
 # force Python 3
 LSST_PYTHON_VERSION=3
-LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.5.4}
+LSST_MINICONDA_VERSION=${LSST_MINICONDA_VERSION:-4.5.12}
 # this git ref controls which set of conda packages are used to initialize the
 # the default conda env defined in scipipe_conda_env git package (RFC-553).
-LSST_SPLENV_REF=${LSST_SPLENV_REF:-${LSST_LSSTSW_REF:-fcd27eb}}
+LSST_SPLENV_REF=${LSST_SPLENV_REF:-${LSST_LSSTSW_REF:-048c331}}
```

Comment:
The SHA1 here might change so make sure you finalize the scipipe_conda_env changes before merging this branch.

Reply:
The actual SHA1 fcd27eb corresponds to a branch commit. 
The new one 048c331 es equivalent, and can be used in the same way.